jp/components/grids_and_lists.md

```diff
@@ -26,175 +26,302 @@ _language: ja
+      <a href="grid_virtualization.md#remote-virtualization" class="feature__more-details">仮想化でリモート ソースを使用</a>
+    </div>
+    <div class="feature__details">
+      <h3 id="remote-data-load-on-demand">[リモート データのロードオンデマンド](grid_virtualization.md#remote-virtualization)</h3>
+      <p>無制限のスクロールが可能でリモート サービスからグリッドのスクロールに合わせてデータを取得するため、あらゆるリモート データ サイズで高いパフォーマンスを発揮します。</p>
+    </div>
+  </div>
+
+  <div class="feature">
+    <div class="feature__image feature__image--right">
+      <img class="b-lazy" src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-lazy-load.png" data-src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-768.gif" data-srcset="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-480.gif 480w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-768.gif 768w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-1100.gif 1100w" sizes="(min-width: 1800px) 575px,(max-width: 1799px) and (min-width: 1200px) 50vw,(max-width: 1199px) and (min-width: 992px) 33vw,(max-width: 991px) and (min-width: 768px) 50vw, 100vw" alt="Paging component allows for programmatic pagination">
       <a href="grid_paging.md" class="feature__more-details">ページング コンポーネント ドキュメント</a>
     </div>
     <div class="feature__details">
-      <h3 id="paging">[ページング](grid_paging.md)</h3>
+      <h3 id="paging">[Paging](grid_paging.md)</h3>
```

Comment:
@yoshimis Can we keep the grid feature names in Japanese?  This happens multiple times in this set of changes.  At least we should match the titles of the topics they link to, and we translate the grid feature topic titles.